bin/Packager/CollectionPackagerBase.py

```diff
@@ -256,6 +262,50 @@ def copyFiles(self, srcDir, destDir, dontStrip) -> bool:
                     utils.sign([entry_target])
         return True
 
+    def symbolsDir(self):
+        """return absolute path to the symbol directory in imageDir
+        """
+        return os.path.join(self.imageDir(), 'symbols')
+
+    def internalPostInstall(self):
+        if not super().internalPostInstall():
+            return False
+
+        if self._doDumpSymbols:
+            for entry in self.traverse(self.imageDir(), self.whitelisted, self.blacklisted):
```

Comment:
We probably won't have a black or white list at the point we execute this code.
Please use os.walk for now or something similar